install/kubernetes/cilium/values.yaml

```diff
@@ -1803,6 +1803,8 @@ enableMasqueradeRouteSource: false
 enableIPv4BIGTCP: false
 # -- Enables IPv6 BIG TCP support which increases maximum IPv6 GSO/GRO limits for nodes and pods
 enableIPv6BIGTCP: false
+# -- Attach endpoint programs using tcx instead of legacy tc hooks on supported kernels.
+enableTCX: true
```

Comment:
nit: Would it make sense to put this under the `bpf` map, i.e. make the value `bpf.enableTCX`?

Reply:
@gandro Done, and I simplified the chart logic a bit (always rendering `enable-tcx` to the configmap), PTAL.

Reply:
Looks good, thanks! Yeah, our charts are super inconsistent unfortunately

Reply:
At some point probably makes sense to bite the bullet and clean it up for more consistency where feasible. Anyway, moving under bpf makes sense. Lgtm.